src/app/destiny1/loadout-builder/calculate.ts

```diff
@@ -117,7 +120,7 @@ export async function getSetBucketsStep(
                       Ghost: ghost,
                     },
                     stats: {
-                      144602215: {
+                      [144602215]: {
```

Comment:
This change is inconsistent with the below entries. Not sure if there's a prettier rule for this?

Reply:
I guess these shouldn't use magic numbers in the first place - 38c553d36f0dfa9a7557ebc09e2df2038c57b6ac uses D1_StatHashes for these, you can can either cherry pick here or I can submit that as a follow up PR.

Reply:
Ha, yeah, this was me rewriting to use `StatHashes`, deciding better of it, and not backing the change all the way out.